pkg/service/user/user.go

```diff
@@ -0,0 +1,96 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+package user
+
+import (
+	"context"
+	"errors"
+	"time"
+
+	"github.com/jackc/pgx/v4"
+
+	"github.com/tensorchord/envd-server/pkg/query"
+)
+
+var (
+	// ExpirationTimeDefault is the default expiration time of the token.
+	expirationTimeDefault = time.Hour * 24 * 365
+	// jwtSecretDefault is the secret used to sign the token.
+	jwtSecretDefault = "envd-secret"
```

Comment:
I think the secret should be generated and should not be leaked. Should be the same for one server, but different for each server

Reply:
I do not think we should generate it. Maybe we can make it a config. WDYT

Reply:
config is fine. I'm not sure whether we need to store this in db

Reply:
Both the JWT secret and expiration time should be configurable. The default one should be much more complex.

I think we don't need to store them in DB.